site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -221,13 +221,109 @@ in [SetLabel(...)](#setlabelkey-value) above. The isolation is also important as
 Setting `GameServer` annotations can be useful if you want information from your running game server process to be 
 observable through the Kubernetes API.
 
+### Counters And Lists
```

Comment:
The player tracking functions `SetCapacity()`, `PlayerConnect()`, and `PlayerDisconnect()` are in the function reference section at the top of functions that change game server state. The counts & lists functions that change state should be added there too.

Reply:
Oh good catch! I will add this right now!

Reply:
...and done!